solr/contrib/ltr/src/java/org/apache/solr/ltr/store/rest/ManagedStoreUtils.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.solr.ltr.store.rest;
+
+import org.apache.solr.core.SolrCore;
+import org.apache.solr.core.SolrResourceLoader;
+import org.apache.solr.rest.ManagedResourceObserver;
+
+public class ManagedStoreUtils {
+
+  // TODO: only package visibility (for test use) for the end points
+
+  /** the feature store rest endpoint **/
+  public static final String FEATURE_STORE_REST_END_POINT = "/schema/feature-store";
+  /** the model store rest endpoint **/
+  public static final String MODEL_STORE_REST_END_POINT = "/schema/model-store";
+
+  public static void registerManagedFeatureStore(SolrResourceLoader solrResourceLoader,
+      ManagedResourceObserver managedResourceObserver) {
+    solrResourceLoader.getManagedResourceRegistry().registerManagedResource(
+        FEATURE_STORE_REST_END_POINT,
+        ManagedFeatureStore.class,
+        managedResourceObserver);
+  }
+
+  public static void registerManagedModelStore(SolrResourceLoader solrResourceLoader,
+      ManagedResourceObserver managedResourceObserver) {
+    solrResourceLoader.getManagedResourceRegistry().registerManagedResource(
+        MODEL_STORE_REST_END_POINT,
+        ManagedModelStore.class,
+        managedResourceObserver);
+  }
+
+  public static ManagedFeatureStore getManagedFeatureStore(SolrCore core) {
```

Comment:
I like the idea of having these utility functions.  However I dont like the idea of having another file that people have to import to get/set managed stores.  Callers of this have to import the ManagedFeature/Model store anyways to hold on to the reference of the resource you return.
You should keep the all the nice helper functions you made, but move them into their respective Managed*Store class and delete this file.